src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -187,16 +187,16 @@ type SynMeasure =
     | Named of longId: LongIdent * range: range
 
     /// A product of two units of measure, e.g. 'kg * m'
-    | Product of measure1: SynMeasure * measure2: SynMeasure * range: range
+    | Product of measure1: SynMeasure * opRange: range * measure2: SynMeasure * range: range
```

Comment:
```suggestion
    | Product of measure1: SynMeasure * asteriskRange: range * measure2: SynMeasure * range: range
```

Reply:
If we want to stick with well-known names, it'd be `mAsterisk`.